test/appium/tests/critical/chats/test_1_1_public_chats.py

```diff
@@ -862,29 +863,65 @@ def prepare_devices(self):
     @marks.testrail_id(702730)
     def test_1_1_chat_message_reaction(self):
         message_from_sender = "Message sender"
-        self.device_1.just_fyi("Sender start 1-1 chat, set emoji and check counter")
+        self.device_1.just_fyi("Sender start 1-1 chat, set 'thumbs-up' emoji and check counter")
         self.chat_1.send_message(message_from_sender)
         self.chat_1.chat_element_by_text(message_from_sender).wait_for_sent_state(120)
         self.chat_1.set_reaction(message_from_sender)
 
         message_sender = self.chat_1.chat_element_by_text(message_from_sender)
         message_sender.emojis_below_message().wait_for_element_text(1)
 
-        self.device_2.just_fyi("Receiver sets own emoji and verifies counter on received message in 1-1 chat")
+        self.device_2.just_fyi(
+            "Receiver also sets 'thumbs-up' emoji and verifies counter on received message in 1-1 chat")
         message_receiver = self.chat_2.chat_element_by_text(message_from_sender)
         message_receiver.emojis_below_message().wait_for_element_text(1, 90)
         self.chat_2.add_remove_same_reaction(message_from_sender)
         message_receiver.emojis_below_message().wait_for_element_text(2)
         message_sender.emojis_below_message().wait_for_element_text(2, 90)
 
-        self.device_2.just_fyi("Receiver pick the same emoji and verify that counter will decrease for both users")
+        self.device_2.just_fyi(
+            "Receiver removes 'thumbs-up' emoji and verify that counter will decrease for both users")
         self.chat_2.add_remove_same_reaction(message_from_sender)
         message_receiver.emojis_below_message().wait_for_element_text(1)
         message_sender.emojis_below_message().wait_for_element_text(1, 90)
+
+        self.device_2.just_fyi("Receiver sets another reaction ('love'). Check it's shown for both sender and receiver")
+        self.chat_2.set_reaction(message_from_sender, emoji="love")
+        message_receiver.emojis_below_message().wait_for_element_text(1)
+        message_sender.emojis_below_message().wait_for_element_text(1, 90)
+        message_receiver.emojis_below_message(emoji="love").wait_for_element_text(1)
+        message_sender.emojis_below_message(emoji="love").wait_for_element_text(1, 90)
```

Comment:
Why it is 2 times here?:
message_receiver.emojis_below_message().wait_for_element_text(1)
message_receiver.emojis_below_message(emoji="love").wait_for_element_text(1)